pytorch_lightning/core/lightning.py

```diff
@@ -214,6 +215,14 @@ def logger(self):
         """ Reference to the logger object in the Trainer. """
         return self.trainer.logger if self.trainer else None
 
+    @property
+    def log_hyperparams(self) -> bool:
+        return self._log_hyperparams
+
+    @log_hyperparams.setter
+    def log_hyperparams(self, log: bool) -> None:
+        self._log_hyperparams = log
```

Comment:
why do we need a property for that if it only forwards it to a protected attribute?

Reply:
Updated!